tests/test_indexer.py

```diff
@@ -268,11 +268,60 @@ def test_subscription_notification_unsuccessful(self):
                          f"WARNING:.*:Failed notification for subscription {subscription_id}"
                          f" for bundle {bundle_id} with transaction id .+ Code: {error_response_code}")
 
-    def test_alias_exists(self):
+    def test_get_index_shape_identifier(self):
+        from dss.events.handlers.index import get_index_shape_identifier
+        index_document = \
+            {
+                'files': {
+                    'assay_json': {
+                        'core': {
+                            'schema_url': "http://hgwdev.soe.ucsc.edu/~kent/hca/schema/assay.json",
+                            'schema_version': "3.0.0",
+                            'type': "assay"
+                        }
+                    },
+                    'sample_json': {
+                        'core': {
+                            'schema_url': "http://hgwdev.soe.ucsc.edu/~kent/hca/schema/sample.json",
+                            'schema_version': "3.0.0",
+                            'type': "sample"
+                        }
+                    }
+                }
+            }
+        self.assertEqual(get_index_shape_identifier(index_document, logger), "v3")
+
+        # Test with mixed/inconsistent metadata schema release versions in the same bundle
```

Comment:
new test or subtest  `with self.subtest("mixed/inconsistent metadata schema release versions in the same bundle"):`